task02_iryna/src/service/parser/ParagraphParser.java

```diff
@@ -0,0 +1,24 @@
+package service.parser;
+
+import dao.CompositeWord;
+import dao.LeafWord;
+import dao.Paragraph;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ParagraphParser extends AbstractParser {
+
+    private static final String SENTENSE_SPLIT_REGEX = "[\\.][\\s]";
+
+    @Override
+    public LeafWord parseLine(String string) {
```

Comment:
I believe that  `SentenseParser`, `TextParser` and `ParagraphParser` could be refactored to one. All you need to send different regexes to it and reuse code every level upper than word.